docs/src/main/resources/microsite/data/menu.yml

```diff
@@ -84,6 +84,10 @@ options:
         url: chisel3/docs/explanations/interfaces-and-connections.html
         menu_type: chisel3
         menu_section: interfaces-and-connections
+      - title: Deep Dive into Connection Operators
```

Comment:
Let's move this to the end of the list, because it doesn't quite make sense before the `DontCare` explanation